### PR TITLE
Read the docs adaptation

### PR DESCRIPTION
**Pull request by FrederickDeny**
Migration of the documentation system from keeping the generated html files tracked to a ReadTheDocs hosted and built documentation

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -165,8 +165,8 @@ completion:
 # Documentation targets and variables
 
 PROJECT=.
-DOCS=$(PROJECT)/doc-source
-MAN=$(PROJECT)/doc-source/build/man
+DOCS=$(PROJECT)/docs
+MAN=$(PROJECT)/docs/build/man
```
Comment (spoutn1k) [inline Makefile:169]: Rename to MAN + BUILDDIR

File: docs/conf.py [hunk 2]
```diff
@@ -13,12 +13,12 @@
 import os
 import sys
 
-sys.path.insert(0, os.path.abspath('../../packages'))
+sys.path.insert(0, os.path.abspath('../packages'))
```
Comment (spoutn1k) [inline docs/conf.py:16]: Use `__file__` to clean the relative import

**Review by spoutn1k — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (FrederickDeny) [reply Makefile]: Done

File: docs/conf.py (hunk 2 above)
Comment (FrederickDeny) [reply docs/conf.py]: Done!

**Review by spoutn1k — APPROVED** [review]

**Pull request merged**